### DISCUSSION TITLE
3.58 two issues noticed

### DISCUSSION THREAD

**Discussion by tampadesignr** [mifi/lossless-cut#1785] [Support] [answered]
1. saving merged files will overwrite even if the number of segments changed.. this did not happen in older version
2. hitting the playback of selected segments does not function like older version it doesnt start at the first segment selected and there is a weird glitch then it starts at last segment selected

**Comment by mifi** [comment]
> * saving merged files will overwrite even if the number of segments changed.. this did not happen in older version

not sure what you mean. what will be overwritten, and how?

> 2\. hitting the playback of selected segments does not function like older version it doesnt start at the first segment selected and there is a weird glitch then it starts at last segment selected

I just tested this with two segments, and it works for me. It plays the segments in order and there is no glitch at the end of the first segment

**Reply by tampadesignr** [reply]
before i could choose a bunch of segments and save then remove a few and save and it would create a new file. now it overwrites.
if i choose the option to not overwrite it will not merge files.

if i create 10 segments and i have selector at last segment and hit the button to rum thru them all it doesnt jump to first and misc other glitches.

**Comment by mifi** [comment]
> before i could choose a bunch of segments and save then remove a few and save and it would create a new file. now it overwrites.

by save do you mean "export"? what kind of file name would it create before as opposed to what it does now?

> if i create 10 segments and i have selector at last segment and hit the button to rum thru them all it doesnt jump to first and misc other glitches.

ok I see the problem. will fix

**Comment by tampadesignr** [comment]
by save do you mean "export"? what kind of file name would it create before as opposed to what it does now?

yes export.
and before it would name each one 
-cut-merged-1704454927870.mp4
then
-cut-merged-1704454941830.mp4
then
-cut-merged-1704454983029.mp4
this was after adding a keyframe it would create a new file which i wanted
now 
-cut-merged-1704455108062.mp4
and i add a keyframe
-cut-merged-1704455108062.mp4
and i add a keyframe
-cut-merged-1704455108062.mp4
meaning it doesn't make a new file
------------------------------------------------
+++ update----just downloaded nightly build and it seems to work like before

**Reply by mifi** [reply, marked as answer]
right, it was a bug, fixed in #1757

**Reply by tampadesignr** [reply]
fyi... 3.59.1.0 still has issue but the latest nightly build did not
